src/main/java/com/redhat/cloud/policies/app/rest/PolicyCrudService.java

```diff
@@ -475,10 +475,11 @@ public Response deletePolicy(@PathParam("id") UUID policyId) {
   }
 
   @Operation(summary = "Delete policies for a customer by the ids passed in the body. Result will be a list of deleted UUIDs")
+  @Parameter(name = "uuids", description = "Array of policies to be deleted", schema = @Schema(type = SchemaType.ARRAY, implementation = UUID.class))
   @APIResponse(responseCode = "403", description = "Individual permissions missing to complete action")
   @APIResponse(responseCode = "200", description = "Policies deleted",
       content = @Content(schema = @Schema(type = SchemaType.ARRAY, implementation = UUID.class)))
-  @DELETE
+  @POST
   @Path("/ids")
```

Comment:
If we are going to change to POST, should be also change the path to `/ids/delete` or similar?